modal/functions.py

```diff
@@ -821,6 +821,7 @@ async def _load(self: _Function, resolver: Resolver, existing_object_id: Optiona
                     warm_pool_size=keep_warm or 0,
                     runtime=config.get("function_runtime"),
                     runtime_debug=config.get("function_runtime_debug"),
+                    runtime_perf_record=config.get("runtime_perf_record"),
```

Comment:
Do we need to also add this to the Config system? https://github.com/modal-labs/modal-client/blob/0cf1e577e21fe9710462fb9c75024ac5b8276e87/modal/config.py#L199-L224

Reply:
Yep, good catch!